doc/examples/joystick/lib/static_joystick.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/joystick.dart';
+import 'package:flame/sprite.dart';
+import 'package:flutter/cupertino.dart';
+
+import 'player.dart';
+
+class StaticJoystick extends Joystick {
+  StaticJoystick() {
+    visible = true;
```

Comment:
I think all these properties should be received by the Joystick constructor as named parameters, as it this is a great way to for people to understand your API